admin_manual/configuration/server/occ_command.rst

```diff
@@ -681,6 +681,11 @@ For example:
   --path="/alice/files/Music"
 
 In the example above, the user_id ``alice`` is determined implicitly from the path component given.
+
+.. note::
+  Mounts are only scanned at the point of creation. Because it is not necessary, mounts recieved like
+  shares or recieved federated shares are not scanned at the recieving user.
```

Comment:
s/recieved/received

silly English

Reply:
I agree with @phil-davis. Additionally, the second sentence doesn't make a lot of sense. I played with it a little, and came up with the following:

> Because it is not necessary, mounts received like shares or that receive federated shares are not scanned by the receiving user.

However, even that doesn't quite make complete sense. @mmattel, can you have another look and either update it or let me know what I'm missing?